.github/pull_request_template.md

```diff
@@ -11,5 +11,3 @@
 Here is some console output
 ```
 - [ ] I ran `make lint` and have made the bulk of changes that it has requested.
-- [ ] The `Lint` GitHub Action step is passing.
-  - Manually run `make format` to correct the errors.
```

Comment:
The `Lint` GH Action must pass for merging to happen at all, so I removed them from the template.